.github/workflows/cli-release.yml

```diff
@@ -28,24 +28,24 @@ jobs:
     runs-on: large-hosted
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Fetch tags required by goreleaser
         run: git fetch --prune --unshallow
 
       - name: Checkout sysroot for cross compilation
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           repository: goreleaser/goreleaser-cross-example-sysroot
           path: sysroot
 
       - name: Set up Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v5
         with:
           go-version: 1.22
 
       - name: Set up NodeJS
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: 18
```

Comment:
Have we been using `node:18` as our main development version since December 2022, or should we be using `node:20` instead?